tests/UnsafeInterfacesTest.php

```diff
@@ -48,4 +58,16 @@ public function testAlwaysValidChild() {
     $xhp = <html>{$x}<body /></html>;
     $this->assertEquals('<html>foo<body></body></html>', $xhp->toString());
   }
-}
+
+  public function testUnsafeAttribute(): void {
+    // without using XHPUnsafeAttributeValue, each &amp; will be double-escaped as &amp;amp;
+    $attr = "document.querySelector('div').innerHTML='foo &amp;&amp; bar'";
+    $xhp = <div onclick={$attr}>click me</div>;
+    $this->assertEquals('<div onclick="document.querySelector(\'div\').innerHTML=\'foo &amp;amp;&amp;amp; bar\'">click me</div>', $xhp->toString());
+
+    // using XHPUnsafeAttributeValue the &amp; is not double escaped
+    $escaped = new ExampleUnsafeAttribute("document.querySelector('div').innerHTML='foo &amp;&amp; bar'");
+    $xhp = <div onclick={$escaped}>click me</div>;
+    $this->assertEquals('<div onclick="document.querySelector(\'div\').innerHTML=\'foo &amp;&amp; bar\'">click me</div>', $xhp->toString());
+  }
+}
```

Comment:
thanks, just nits then :)

- newline at end of file
- remove extra newline at 36
- this can be demonstrated without the query selector to make it shorter - just the foo...bar part. Doesn't need to be valid JS, just to deomnstrate the escaping